packages/api/cms-api/src/blocks/decorators/field.ts

```diff
@@ -32,6 +32,11 @@ type BlockFieldOptions =
           block: Block;
       };
 
+/** @knipignore */
+export const BLOCK_FIELD_METADATA_KEY = "data:fieldType";
+/** @knipignore */
+export const BLOCK_FIELD_KEYS_METADATA_KEY = "keys:field";
```

Comment:
```suggestion
const BLOCK_FIELD_METADATA_KEY = "data:fieldType";
const BLOCK_FIELD_KEYS_METADATA_KEY = "keys:field";
```

What do you think about not exporting the constants instead of disabling Knip? We can export them once they're used in another location.

Reply:
is ok for me done